### ISSUE TITLE
Continuous testing

### ISSUE THREAD

**Issue by fredrikbk** [tensor-compiler/taco#157]
We need a continuous testing server to ensure taco compiles and works with different compiler versions on different platforms (gcc, clang, linux, macos). It should run tests every time we commit to master and send emails if something fails.

For example, we can set up a Jenkins server on CSAIL OpenCloud.

**Comment by Ravenwater** [comment]
are you open to use some of the open source CI platforms (travis, codeship, appveyor, etc.?)

**Comment by fredrikbk** [comment]
Definitely, I'm not particular to any platform. Which one do you recommend? We'll need to set up git commit hooks and to compile and test remotely (through SSH or something similar).

**Comment by Ravenwater** [comment]
One of our contributors on the posit number system suggested AppVeyor as it supports Windows builds. I haven't build any CI pipelines with AppVeyor so can't judge if it is productive or not. We are using CodeShip as they have a nice container workflow, but CodeShip doesn't have the Windows support, so I feel that AppVeyor might be the better path.

**Comment by Ravenwater** [comment]
just tried AppVeyor to create a Windows build for stillwater-sc/universal and it is pretty straight forward. I  like the web UI, very informative. So, AppVeyor is a good candidate. I'll try to get sharp on it.
